apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/RefineModal/index.tsx

```diff
@@ -1,5 +1,3 @@
-'use client'
-
 import { ReactNode, useEffect, useMemo, useState } from 'react'
 
 import {
```

Comment:
How is it not a client component if it uses `useEffect`, `useMemo` and `useState`?